app/experimenter/nimbus-ui/src/components/AppLayoutWithSidebar/index.tsx

```diff
@@ -118,6 +119,8 @@ export const AppLayoutWithSidebar = ({
                   {page.name}
                 </LinkNav>
               ))}
+
+              <SidebarActions experiment={experiment} />
```

Comment:
Gonna channel my inner @LZoog and say you can provide this prop like `{...{ experiment }}`